### DISCUSSION TITLE
How is the theme pulling Featured?

### DISCUSSION THREAD

**Discussion by toddgrotenhuis** [endonend/Hains-Point#18] [Q&A]
I setup autofilter to mark all titled posts with the Featured category and all micro posts as Asides category. You can see it worked:

<img width="425" alt="image" src="https://user-images.githubusercontent.com/165077/157508572-5da98ce8-068f-4be9-a5ac-284c64f997a4.png">

At times when I was setting up my blog on the new theme (and editing settings), the latest two Featured posts would show, and at other times none would show. Now today, a post I moved to my blog (from an old system) but backdated to 2016 is the only one showing in featured.

<img width="585" alt="image" src="https://user-images.githubusercontent.com/165077/157508738-8f3adbc9-640b-4949-b1dc-f85d503a64e9.png">

**Comment by toddgrotenhuis** [comment]
I've just refreshed after update to newest version and now I have the 2 latest full posts showing again. Maybe the microformats are related?

<img width="949" alt="image" src="https://user-images.githubusercontent.com/165077/157509674-3ae9e9d6-cd15-4fe1-a9d6-debf1486fd15.png">

**Reply by endonend** [reply]
Glad that worked! I'd guess things were cached on the MB end, especially if you made bulk changes to posts to put them in categories. See my note below on using the rebuild button if you run into things that could possibly fit in the caching category.

**Comment by endonend** [comment]
The code should pull the first two (most recent) posts in the Featured category:

> {{ with .Site.Taxonomies.categories.featured }}
> 		{{ range first 2 . }}

Early on in the development, sometimes one post would show, sometimes two... once I stopped making major changes to the homepage view, it settled. One thing you can try is to trigger a full rebuild of your site... to do that, go to https://micro.blog/account and scroll down to bottom and click on "View logs"... from that page, click the Rebuild button at top. 

I'd guess that would clean out any weirdness/cached file/etc...

**Comment by toddgrotenhuis** [comment]
I am on my second rebuild today, to get it back to pulling the right posts. It seems like when I make a tweak on the settings page it reverts to pulling the wrong things. I haven't been able to figure it out entirely.
